src/bots/uncrossArbBot.ts

```diff
@@ -324,4 +572,70 @@ export class UncrossArbBot implements Bot {
 			}
 		}
 	}
+
+	private async settlePnls() {
```

Comment:
we should recursively try to call settle pnls in smaller chunks if they fail i think ... we should probably generalize that pattern to so we don't have to keep implementing it, i can do that